flow/Makefile

```diff
@@ -371,15 +373,19 @@ versions.txt:
 # Pre-process libraries
 # ==============================================================================
 
-# Create temporary Liberty files which have the proper dont_use properties set
-# For use with Yosys and ABC
+# Unzip any compressed Liberty files
 .SECONDEXPANSION:
-$(DONT_USE_LIBS): $$(filter %$$(@F) %$$(@F).gz,$(LIB_FILES))
-	@mkdir -p $(OBJECTS_DIR)/lib
-	$(UTILS_DIR)/markDontUse.py -p "$(DONT_USE_CELLS)" -i $^ -o $@
+$(DECOMPRESSED_LIBS): $$(filter %$$(@F) %$$(@F).gz,$(LIB_FILES))
+	mkdir -p $(OBJECTS_DIR)/lib
+	if [[ $^ == *.gz ]]; then \
+	  gunzip -c $^ > $@; \
+	else \
+	  ln -s $$(realpath --relative-to=$(dir $@) $^) $@; \
+	fi
```

Comment:
there is one additional issue here.
You still need to use the mark don't use script to ensure the yosys call to map dffs doesn't use any FFs that are marked as don't use.
Mark don't use also does some additional cleanup which has caused ABC to crash before on other PDKs.

In order to actually get rid of this, you will need:
1. yosys support for adding don't use to the dff call in that script (otherwise it's a loss of functionality).
2. abc and yosys some improvements to their liberty parsers to not choke on pdks. https://github.com/siliconcompiler/siliconcompiler/blob/55d8ff05d42f55ac45970641a51f0817a9918b3f/siliconcompiler/tools/yosys/prepareLib.py#L43 https://github.com/siliconcompiler/siliconcompiler/blob/55d8ff05d42f55ac45970641a51f0817a9918b3f/siliconcompiler/tools/yosys/prepareLib.py#L51 https://github.com/siliconcompiler/siliconcompiler/blob/55d8ff05d42f55ac45970641a51f0817a9918b3f/siliconcompiler/tools/yosys/prepareLib.py#L58